File: Kanban-ASP.NET/Kanban-BE/Kanban.Model/Models/TaskModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Kanban.Model.Models
{
    public class TaskModelToCreate
    {
        public string Name { get; set; }
        public int? OwnerId { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? DueDate { get; set; }
        public string TaskStatuts { get; set; }
        public int? Priority { get; set; }
        public int? Completion { get; set; }
        public int? AssignedEmployeeId { get; set; }
        public double? IndexTask { get; set; }
        public int ListId { get; set; }
    }
}

File: Kanban-ASP.NET/Kanban-UI/src/app/models/task.model.ts
export class Task {
  id: number;
  name: string;
  ownerId: number;
  startDate: Date;
  dueDate: Date;
  taskStatuts: string;
  priority: number;
  completion: number;
  assignedEmployeeId: number;
  indexTask: number;
  listId: number;
  assignedEmployee: AssignedEmployee;
  owner: Owner;
}

export class AssignedEmployee {
  id: number;
  name: string;
}

export class Owner {
  id: number;
  name: string;
}
